src/components/Search/SearchStatusBar.tsx

```diff
@@ -126,10 +128,16 @@ function SearchStatusBar({type, status}: SearchStatusBarProps) {
     const theme = useTheme();
     const {translate} = useLocalize();
     const options = getOptions(type);
+    const {shouldShowStatusBarLoading} = useSearchContext();
+
+    /** We only want to display the skeleton for the status filters the first time we load them for a specific data type */
```

Comment:
I think a better place for this comment is inside context, as the comment describing this specific variable

Reply:
or in the place where the value of the context is being set - your choice